code/modules/mob/living/carbon/xenomorph/castes/gorger/abilities_gorger.dm

```diff
@@ -197,7 +198,7 @@
 
 	if(owner.do_actions)
 		return FALSE
-	if(!line_of_sight(owner, target_xeno, 2) || get_dist(owner, target_xeno) > 2)
+	if(!line_of_sight(owner, target_xeno, 3) || get_dist(owner, target_xeno) > 3)
```

Comment:
I know this was existing code, but again, I am fairly sure line_of_sight fails if they're outside of the specified range anyway.